sdk/storage/Azure.Storage.Files/tests/FileTestBase.cs

```diff
@@ -123,39 +123,48 @@ public StorageSharedKeyCredential GetNewSharedKeyCredentials()
                 TestConfigDefault.AccountKey);
 
         public SasQueryParameters GetNewAccountSasCredentials(StorageSharedKeyCredential sharedKeyCredentials = default)
-            => new AccountSasBuilder
+        {
+            var builder = new AccountSasBuilder
             {
                 Protocol = SasProtocol.None,
                 Services = AccountSasServices.Files,
                 ResourceTypes = AccountSasResourceTypes.Container,
                 StartsOn = Recording.UtcNow.AddHours(-1),
                 ExpiresOn = Recording.UtcNow.AddHours(+1),
-                Permissions = new FileAccountSasPermissions { Create = true, Delete = true }.ToString(),
                 IPRange = new SasIPRange(IPAddress.None, IPAddress.None)
-            }.ToSasQueryParameters(sharedKeyCredentials);
+            };
+            builder.SetPermissions(AccountSasPermissions.Create | AccountSasPermissions.Delete);
```

Comment:
You had `FileAccountSasPermissions` but changed it to `AccountSasPermissions`

Reply:
It is using an AccountSasBuilder though, so I think it was incorrect before.